pkg/controller/embercsi/embercsi_controller.go

```diff
@@ -491,12 +487,8 @@ func getVolumeMounts(ecsi *embercsiv1alpha1.EmberCSI, csiDriverMode string) []co
 			Name: "iscsi-dir",
 			MountPropagation: &bidirectional,
 		},{
-			MountPath: "/etc/lvm",
```

Comment:
-1: This will break the LVM backend, we should add it conditionally depending on the driver being used.